src/core.js

```diff
@@ -1,5 +1,6 @@
 
 var oldInit = jQuery.fn.init,
+	oldIsNumeric = jQuery.isNumeric,
```

Comment:
Dunno why I did this, I could have called it directly since I'm not patching it. :smile_cat: